objects/src/accounts/mod.rs

```diff
@@ -68,14 +68,10 @@ pub const ACCOUNT_ID_INSUFFICIENT_ONES: u64 = 0b1100000110 << 54;
 /// changed). Other components may be mutated throughout the lifetime of the account. However,
 /// account state can be changed only by invoking one of account interface methods.
 #[derive(Debug, Clone, PartialEq, Eq)]
-#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
```

Comment:
The risk with this new implementation is that we forget to update the serialization code if we change the `Account` struct. This can be mitigated by rewriting `write_into()` as:

```rs
impl Serializable for Account {
    fn write_into<W: ByteWriter>(&self, target: &mut W) {
        // compile error here if `Account` changes
        let Account { id, vault, storage, code, nonce } = self;

        id.write_into(target);
        vault.write_into(target);
        storage.write_into(target);
        code.write_into(target);
        nonce.write_into(target);
    }
}
```

If we change `Account`, then the `Account` destructuring statement will cause a compile error. We should probably adopt this everywhere (until we have a derive macro for `Serializable`)